dtcm/extraterrestrial/map.xml

```diff
@@ -1,8 +1,14 @@
 <map proto="1.4.2">
 <name>Extraterrestrial</name>
-<version>1.1.3</version>
+<variant id="christmas" world="christmas" override="true">Extraterrestrial: Sub-Zero</variant>
```

Comment:
No need for ``override="true"`` here. Variant name can just be ``Sub-Zero``.

Reply:
> No need for `override="true"` here. Variant name can just be `Sub-Zero`, and it'll produce the same effect.

@zzufx fixed in new commit